axelrod/tests/strategies/test_axelrod_second.py

```diff
@@ -2034,7 +2034,7 @@ def test_strategy(self):
 
 class TestSeconodByDowning(TestPlayer):
```

Comment:
Seconod -> Second

Are we sure we don't want to call this one RevisedDowning ? (Here is where the classifier would separate the naming concerns from the inclusion in the first or second tournament...)

Reply:
Yup, that'll be similar to TitForTat, Grudger etc... I'll change it.

Reply:
a6471e8 renames this and moves it to its own file so that the description at the top of `axelrod_second.py` is still accurate.